Sources/MapFilter/StepValue.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+struct StepValue<ValueKind: Comparable>: Equatable, Comparable, SliderReferenceValue {
```

Comment:
Since we have the referenceIndex array as an argument I would probably use that directly instead of using this structure. It is easier to see which values are referenced etc.

Reply:
Hmm, yes maybe, since I had to make a "formatter" class to pass around anyway in the end it might be possible to skip this class. Will look into it 👍

Reply:
I have removed `StepValue` now 👍